src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -9,6 +9,7 @@
     <!-- BUILDING_INBOX_LIBRARY is only false when building for netstandard to validate that the sources are netstandard compatible. -->
     <!-- This is meant to help with producing a source package and not to ship a netstandard compatible binary. -->
     <DefineConstants Condition="'$(TargetsNETStandard)' != 'true'">$(DefineConstants);BUILDING_INBOX_LIBRARY</DefineConstants>
+    <DefineConstants Condition="'$(TargetsNETStandard)' == 'true'">$(DefineConstants);USE_ABW_INTERNALLY</DefineConstants>
```

Comment:
If this is required for NetStandard, keep in mind that the source package won't have this constant defined, so consumers of the source package will get the netstandard sources, but won't have this constant defined.

Reply:
Good point. It doesn't have any concerns here since all we are flipping here is going from public to internal (which the source package does anyway). But, I will invert the condition. @stephentoub provided the same feedback here: https://github.com/dotnet/corefx/pull/36961#discussion_r276450887